docs/bin/dump_keywords.py

```diff
@@ -46,7 +46,7 @@
     oblist[name] = dict((x, aobj.__dict__['_attributes'][x]) for x in aobj.__dict__['_attributes'] if 'private' not in x or not x.private)
 
     # pick up docs if they exist
-    for a in oblist[name]:
+    for a in list(oblist[name]):
```

Comment:
This has already been fixed in 2.9 and devel

Reply:
If we want it to work on 2.8 and previous we'll need something like this there.  However, 2.8 is soon going to be in critical fixes only, so perhaps this is not needed there.